### PR TITLE
Towards 2.0

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: src/Node/TransNode.php [hunk 1]
```diff
@@ -1,11 +1,156 @@
 <?php
 
+/*
+ * This file is part of Twig.
```
Comment (stof) [inline src/Node/TransNode.php]: shouldn't this say `Twig-extensions` instead ?

File: src/DateExtension.php [hunk 2]
```diff
@@ -1,11 +1,96 @@
...
-    class DateExtension extends \Twig_Extensions_Extension_Date
+        return '';
+    }
+
+    private function getPluralizedInterval($count, $invert, $unit)
     {
+        if ($this->translator) {
+            $id = sprintf('diff.%s.%s', $invert ? 'in' : 'ago', $unit);
+
+            return $this->translator->transChoice($id, $count, ['%count%' => $count], 'date');
```
Comment (endelwar) [inline src/DateExtension.php]: The "Symfony\Component\Translation\Translator::transChoice()" method is deprecated since Symfony 4.2

Comment (stof) [reply src/DateExtension.php]: that should be handled separately though, as even 1.x should skip that.

File: .php_cs.dist [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                // generated fixtures
+                'Symfony/Component/VarDumper/Tests/Fixtures',
+                'Symfony/Component/VarExporter/Tests/Fixtures',
+                // resource templates
+                'Symfony/Bundle/FrameworkBundle/Resources/views/Form',
+                // explicit trigger_error tests
+                'Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/',
+            ])
+            // Support for older PHPunit version
+            ->notPath('Symfony/Bridge/PhpUnit/SymfonyTestsListener.php')
```
Comment (sstok) [inline .php_cs.dist]: This doesn't seem related to this package, and can be removed.
